### PR TITLE
[DOCS-7799]  Network Path - updated PR

### PR DESCRIPTION
**Pull request by aliciascott**
### What does this PR do? What is the motivation?
Adding private beta docs for Network Path - replaces https://github.com/DataDog/documentation/pull/23404

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/network_monitoring/network_path/setup.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Network Path for Datadog Network Performance Monitoring is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+<div class="alert alert-info">Network Path for Datadog Network Performance Monitoring is in private beta. Reach out to your Datadog representative to sign up, and then use the following instructions to configure the Datadog Agent to gather network path data.</div>
+
+
+## Setup Instructions on Linux Agent
+
+1. Agent version `7.55` or higher is required.
```
Comment (alai97) [inline content/en/network_monitoring/network_path/setup.md:24]:
```suggestion
## Overview

Setting up Network Path involves configuring your Linux environment to monitor and trace the network routes between your services and endpoints. This helps identify bottlenecks, latency issues, and potential points of failure in your network infrastructure.

## Setup

Agent version `7.55` or higher is required.


```

File: content/en/network_monitoring/network_path/setup.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+2. Enable [NPM][1].
+3. Enable the `system-probe` traceroute module in `/etc/datadog-agent/system-probe.yaml` by adding the following:
+
+```
+traceroute:
+  enabled: true
+```
```
Comment (alai97) [inline content/en/network_monitoring/network_path/setup.md:31]: Indent this three spaces to the right.

File: content/en/network_monitoring/network_path/setup.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+```
+Example:
+
+network_path:
+  connections_monitoring:
+    enabled: true
+  collector:
+    workers: 10 # default 4
+```
```
Comment (alai97) [inline content/en/network_monitoring/network_path/setup.md:43]: Indent this three spaces to the right.

File: content/en/network_monitoring/network_path/setup.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    ## Enable network path collection
+    #
+    enabled: true
+  collector:
+    ## @param workers - int - optional - default:4
+    ## Number of workers that can collect paths in parallel
+    ## Recommendation: leave at default
+    #
+    workers: 10
+```
```
Comment (alai97) [inline content/en/network_monitoring/network_path/setup.md:61]: Indent this three spaces to the right.

File: content/en/network_monitoring/network_path/setup.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+traceroute:
+  enabled: true
+```
+
+4.  Enable network_path to monitor NPM connections by creating/editing the `/etc/datadog-agent/datadog.yaml` file: 
+
+```
+Example:
+
```
Comment (alai97) [inline content/en/network_monitoring/network_path/setup.md:37]:
```suggestion
```

File: content/en/network_monitoring/network_path/setup.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+1. Agent version `7.55` or higher is required.
+2. Enable [NPM][1].
+3. Enable the `system-probe` traceroute module in `/etc/datadog-agent/system-probe.yaml` by adding the following:
+
+```
+traceroute:
+  enabled: true
+```
+
+4.  Enable network_path to monitor NPM connections by creating/editing the `/etc/datadog-agent/datadog.yaml` file:
```
Comment (alai97) [inline content/en/network_monitoring/network_path/setup.md:33]:
```suggestion
4.  Enable `network_path` to monitor NPM connections by creating or editing the `/etc/datadog-agent/datadog.yaml` file: 
```

File: content/en/network_monitoring/network_path/setup.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+
+network_path:
+  connections_monitoring:
+    enabled: true
+  collector:
+    workers: 10 # default 4
+```
+ 
+
+Detailed available configuration:
```
Comment (alai97) [inline content/en/network_monitoring/network_path/setup.md:46]:
```suggestion
For full configuration details, see the following:
```

File: content/en/network_monitoring/performance/_index.md [hunk 8]
```diff
@@ -48,6 +48,8 @@ Datadog Network Performance Monitoring (NPM) gives you visibility into your netw
 
 NPM makes it simple to monitor complex networks with built in support for Linux and [Windows OS][3] as well as containerized environments that are orchestrated and [instrumented with Istio service mesh][4].
 
+Additionally, [Network path][5], a feature of NPM, is available in private beta, which allows you to see hop by hop traffic in your network.
```
Comment (alai97) [inline content/en/network_monitoring/performance/_index.md:51]:
```suggestion
Additionally, [Network path][5], a feature of NPM, is available in private beta, which allows you to see hop-by-hop traffic in your network.
```

**Review by alai97 — COMMENTED** [review]
Looks good, some copy suggestions.

**Comment by aliciascott** [comment]
Hey @alai97 thanks for the review! Fixed all your suggestions, can you re-review when you get a chance ?

File: content/en/network_monitoring/network_path/path_view.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+Traversal completion 
+: Represents whether or not the `traceroute` was able to successfully reach the destination.
+
+Reachability
+: The level of packet loss the destination is experiencing.
+
+Latency 
+: How long the `traceroute` took to get from a source to its destination.
+
+**Note**: Hop to hop latency may show `N/A` for hops that were incomplete.
```
Comment (alai97) [inline content/en/network_monitoring/network_path/path_view.md:49]:
```suggestion
**Note**: Hop-to-hop latency may show `N/A` for hops that were incomplete.

```

**Pull request merged**